scripts/newmoose.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
```

Comment:
Especially since this is most important for new users, it seems like the most (boring) appropriate approach would be to have something descriptive.  I don't have a strong opinion though and am definitely open to suggestions from everyone.